test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -455,6 +455,9 @@ private void Execute(string args, out string stdOut, out string stdError, out in
                 Console.WriteLine("IntegrationTestBase.Execute: Path = {0}", vstestconsole.StartInfo.FileName);
                 Console.WriteLine("IntegrationTestBase.Execute: Arguments = {0}", vstestconsole.StartInfo.Arguments);
 
+                Stopwatch stopwatch = new Stopwatch();
```

Comment:
wrap Stopwatch inside a class which implements IDisposable pattern. in Dispose stop and print the time. for reference please check with me

Reply:
As discussed offline, IDisposable pattern very interesting. We have few places where we are using stopwatch like this. I'm scoping this PR to just add stopwatch in acceptance tests.